sdk/core/src/index.ts

```diff
@@ -655,42 +491,29 @@ var ReactMoE = {
    * @param {String} pushToken
    */
   passPushKitPushToken: function (pushToken: string) {
-    commonValidationCheck();
     MoEngageLogger.verbose("Will process push-kit push token");
-    let payload = getMoEPushTokenJson(pushToken, PUSH_SERVICE_PUSH_KIT, PLATFORM_ANDROID, moeAppId);
-
     if (Platform.OS == PLATFORM_ANDROID) {
-      MoERNAndroid.passPushKitPushToken(payload);
-    } else {
-      MoEngageLogger.debug("This api is not supported on iOS platform.");
+      let payload = getMoEPushTokenJson(pushToken, PUSH_SERVICE_PUSH_KIT, PLATFORM_ANDROID, moeAppId);
+      MoEReactBridge.passPushKitPushToken(payload);
```

Comment:
lets's add back the debug log for not-supported

Reply:
@umangmoe done